package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "SAML-tracer",
+  "main": "index.js",
+  "repository": "https://github.com/simplesamlphp/SAML-tracer",
+  "license": "LGPL-2.1-or-later",
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=6.0.0"
+  },
+  "scripts": {
+    "postinstall": "minimum-node-version || true",
+    "build": "webpack"
+  },
+  "dependencies": {
+    "highlight.js": "^11.9.0",
+    "minimum-node-version": "^3.0.0",
+    "pako": "^2.1.0"
+  },
+  "devDependencies": {
```

Comment:
Another maybe silly question:

All those dev dependencies were introduced with this commit: https://github.com/simplesamlphp/SAML-tracer/pull/87/commits/bf01bed92fa5b7011c908000bea7f7b67674e549
Hence the question: Is all this stuff required for manually building hljs with our small subset of required language-highlighting?

You certainly hear out: Personally, I'm more of a dependency purist.
That means: If you don't absolutely need it, I would rather do without it. Is that possible? Would probably shrink the package-lock.json, too 😉

Reply:
> Hence the question: Is all this stuff required for manually building hljs with our small subset of required language-highlighting?

Yep, I don't see any other way to do this. Again, the lock-file should just be removed from the repo